tests/integration_tests/test_experiment.py

```diff
@@ -28,6 +28,9 @@
 from tests.integration_tests.utils import ENCODERS
 from tests.integration_tests.utils import generate_data
 from tests.integration_tests.utils import model_definition_template
+# The following imports are pytest fixtures, required for running the tests
+from tests.fixtures.filenames import csv_filename
+from tests.fixtures.filenames import yaml_filename
```

Comment:
Is this used anywhere?

Reply:
yeah, every test is using the csv_filename (pytest fixture). Essentially a random filename is generated for each test and the file is deleted after the test runs.